OpenRA.Game/Server/Server.cs

```diff
@@ -169,14 +108,14 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData)
 
 					foreach (var s in checkRead)
 						if (s == listener.Server) AcceptConnection();
-						else if (PreConns.Count > 0)
+						else if (preConns.Count != 0)
```

Comment:
I could also say this is another reason why `!=` is not needed. The value is always >= 0.
This change just creates redundant diff noise while actually complicating a check that doesn't need to be complicated.